utils/sysinfo/sysinfo.go

```diff
@@ -523,3 +523,14 @@ func GetSocketFromCPU(topology []info.Node, cpu int) int {
 	}
 	return -1
 }
+
+// GetOnlineCPUs returns available cores.
+func GetOnlineCPUs(topology []info.Node) []int {
```

Comment:
May you add unit tests for this function to [this](https://github.com/google/cadvisor/blob/master/utils/sysinfo/sysinfo_test.go) file?